tests/acceptance/pageObjects/personalPage.js

```diff
@@ -418,6 +408,9 @@ module.exports = {
     fileUploadProgress: {
       selector: '#upload-info'
     },
+    fileUploadStatus: {
+      selector: '.upload-info-status .upload-info-success, .upload-info-status .upload-info-danger'
```

Comment:
Should I check for either success or fail selector and make the upload function pass on (_done above_) and let the **Then** step assert the result?
Or, make strict checks for success inside the upload function and create separate `tries to upload` step for tests like [this](https://drone.owncloud.com/owncloud/web/26312/51/20) (where upload should fail)?

Reply:
the `When` step should not concern with whether the upload succeeded or failed

Reply:
I think having a general selector is OK